### DISCUSSION TITLE
CoinStake Abandoned

### DISCUSSION THREAD

**Discussion by b2bfun** [pocketnetteam/pocketnet.core#690] [Q&A]
Hello Guys,

Could you help me for understanding, I've installed Linux Full Node and received a few normal rewards and 2 Abandoned transactions for a month. What should I do with that or what options do I have to fix it or how to avoid it in a future?

**Comment by andyoknen** [comment]
Hi! This is a regular situation - since staking is decentralized, different nodes can create the same block at the same time, but the network can accept only one block. If at the same time you do not notice a decrease in the balance, then you can simply ignore the abandon transaction

**Reply by b2bfun** [reply]
Thank you!
